extra_data/tests/mockdata/base.py

```diff
@@ -142,9 +147,27 @@ def write_metadata(h5file, data_sources, chunksize=16, format_version='0.5'):
 
     if format_version != '0.5':
         h5file['METADATA/dataFormatVersion'] = [format_version.encode('ascii')]
-
-def write_train_ids(f, path, N, first=10000, chunksize=16):
-    """Make a dataset of fake train IDs at the given path
+        now = datetime.utcnow().replace(microsecond=0).strftime('%Y%m%dT%H%M%SZ')
+        h5file['METADATA/creationDate'] = [now.encode('ascii')]
+        h5file['METADATA/daqLibrary'] = [b'1.9.0']
+        h5file['METADATA/karaboFramework'] = [b'2.7.0']
+        h5file.create_dataset('METADATA/proposalNumber', dtype=np.uint32, data=[700000])
+        h5file.create_dataset('METADATA/runNumber', dtype=np.uint32, data=[1])
+        h5file['METADATA/runType'] = [b'Test DAQ']
+        h5file['METADATA/sample'] = [b'No Sample']
+        print(osp.basename(h5file.filename))
+        # get sequence number
+        fname_pattern = r'^(RAW|CORR)\-R\d+\-.*\-S(\d+).h5$'
+        match = re.match(fname_pattern, osp.basename(h5file.filename))
+        sequence = int(match.groups()[1]) if match is not None else 0
+        h5file.create_dataset('METADATA/sequenceNumber', dtype=np.uint32, data=[sequence])
+        h5file['METADATA/updateDate'] = [now.encode('ascii')]
```

Comment:
Interestingly, in some files I've just looked at, `updateDate` is about 5 minutes after `creationDate`. I guess it's the start and end of writing that file. I don't think we need to simulate that, but it might be good to know.

These fields also seem to relate to the file, not the run as a whole; at least, they differ for different files.

Reply:
Interesting, so it's probably when the file is closed? I updated the value to be a few minutes after creationDate